### PR TITLE
Update dialogue design

### PR DESCRIPTION
**Pull request by nunogois**
https://linear.app/unleash/issue/2-509/update-dialogues-to-match-new-designs

Update dialogues to match the look on the new designs:
image

### CODE REVIEW

**Comment by NicolaeUnleash** [comment]
@nunogois I found some dialogs that needs to be fixed first

File: frontend/src/component/common/Dialogue/Dialogue.tsx [hunk 1]
```diff
@@ -5,12 +5,42 @@ import {
...
 
 import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
-import { useStyles } from './Dialogue.styles';
 import { DIALOGUE_CONFIRM_ID } from 'utils/testIds';
 
+const StyledDialog = styled(Dialog)(({ theme, maxWidth }) => ({
+    '& .MuiDialog-paper': {
+        borderRadius: theme.shape.borderRadiusLarge,
+        maxWidth: !maxWidth ? theme.spacing(85) : undefined,
+        backgroundColor: 'transparent',
```
Comment (nunogois) [inline frontend/src/component/common/Dialogue/Dialogue.tsx:18]: This fixes the issue with the extra white pixels at the borders when using border radius.

**Comment by nunogois** [comment]
Checked with @NicolaeUnleash and it seems like this now looks correct!

**Review by FredrikOseberg — APPROVED** [review]
LGTM

**Review by gastonfournier — APPROVED** [review]
G!

**Pull request merged**